hooks.py

```diff
@@ -51,9 +51,10 @@ def notices_updated(notices):
         }
         files = []
         if 'attachment_url' in notice:
-            r = requests.get(notice['attachment_url'], stream=True,
-                             **req_args)
-            r.raw.decode_content = True
+            r = notice['attachment_raw']
```

Comment:
You got it absolutely right. They've shifted from a non-SSO to an SSO URL, and now require auth to read the PDFs. So, `session.get` works whereas `requests.get` doesn't (it used to, previously).

So, I'm storing the raw attachment text to the `notice` variable (db?) and using it while calling the mailer function.

Reply:
AH! I see, I understand your approach now! I was reading the code in the reverse order, because I was confused about the control flow. Thanks for explaining! :)